example/model_usage.ipynb

```diff
@@ -222,52 +216,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "print(f\"The model '{model.name}' has the following properties and metadata\\n\")\n",
-    "print(f\" Description:\")\n",
-    "pprint(model.description)\n",
-    "\n",
-    "print(\"\\n The authors of the model are: \")\n",
-    "pprint(model.authors)\n",
-    "\n",
-    "print(\"\\n It is maintained by:\")\n",
-    "pprint(model.maintainers)\n",
-    "\n",
-    "pprint(\"\\n License: \")\n",
-    "pprint(model.license)\n",
-    "\n",
-    "print(\"\\n If you use this model, you are expected to cite it as: \")\n",
-    "pprint(model.cite)\n",
-    "\n",
-    "print(f\"\\n Further documentation can be found here: {model.documentation.absolute()}\")\n",
-    "\n",
-    "if model.git_repo == None:\n",
-    "    print(f\"\\nNo associated GitHub repository.\")\n",
-    "else:\n",
-    "    print(f\"\\n GitHub repository: \")\n",
-    "    pprint(model.git_repo)\n",
-    "\n",
-    "print(f\"\\n Covers of the model '{model.name}' are: \")\n",
-    "for cover in model.covers:\n",
-    "    cover_data = imread(download(cover).path)\n",
-    "    plt.figure(figsize=(10, 10))\n",
-    "    plt.imshow(cover_data)\n",
-    "    plt.xticks([])\n",
-    "    plt.yticks([])\n",
-    "    plt.show()"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "### 1.4 Get expected model inputs and outputs"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "Each model expects Inputs and returns Outputs with specific features. The following code shows how to get the full metadata of the expected inputs and outputs of the model."
+    "pprint(model)"
```

Comment:
I followed the `4.4 Inspect it all!` way of [spec notebook](https://github.com/bioimage-io/spec-bioimage-io/blob/main/example/load_model_and_create_your_own.ipynb)